Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -252,7 +254,10 @@ def urlify_hyperlinks(message: str) -> str:
     # URLify markdown hyperlinks
     urls = re.findall(URL_REGEX, message)
     for url in urls:
-        formatted_message = formatted_message.replace(url, f'[{url}]({url})')
+        if EXTERNAL_FORM in url:
+            formatted_message = formatted_message.replace(url, f'[Microsoft Teams Form]({url})')
```

Comment:
Since this is a special use case, please add a short docstring to explain where the external_form is coming from.